docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -173,6 +173,17 @@ If you experience any, then please contact Couchbase Technical Support.
 
 *Further Reading*: https://blog.couchbase.com/couchbase-dcp-rollback-qa-tests/[Couchbase DCP Rollback]
 
+[#CB90070]
+=== Permission Denied Errors (CB90070)
+
+*Background*: Couchbase Server has been denied permission to access resources.
+This is potentially due to other applications locking Couchbase Server files, or misconfiguration.
+
+*Condition*: Found permission denied errors in memcached.log.
+
+*Remediation*: Check that there are no other applications locking files in your Couchbase Server directory.
```

Comment:
Permission Denied could also be simply that the CB data directory is owned by a user other than the CB user, not necessarily to do with locking.

Reply:
That's true, how does "Check that there are no other applications locking files in your Couchbase Server directory, and that permissions are correctly configured" sound?